### PR TITLE
Optimization test for convolution

### PR DESCRIPTION
**Pull request by laurent-dinh**

### CODE REVIEW

File: theano/sandbox/cuda/tests/test_abstractconv.py [hunk 1]
```diff
@@ -228,14 +249,19 @@ def test_gpucorrmm_conv(self):
             self.run_fwd(inputs_shape=i, filters_shape=f, subsample=s,
                          verify_grad=True, mode=mode, device='gpu',
                          provide_shape=provide_shape, border_mode=b,
-                         filter_flip=flip)
+                         filter_flip=flip,
+                         target_op=(GpuCorrMM,
+                                    GpuCorrMM_gradWeights,
+                                    GpuCorrMM_gradInputs))
```
Comment (lamblin) [inline theano/sandbox/cuda/tests/test_abstractconv.py]: Good, I did not remember these cases.

**Comment by lamblin** [comment]
Cool, thanks!
OK to merge when Travis passes.

**Comment by lamblin** [comment]
Fix #3668

**Comment by nouiz** [comment]
I killed travis to have another priority PR get merged. When it get merge, we can start travis again.

**Comment by lamblin** [comment]
Travis is actually failing because of PEP8 errors.
@laurent-dinh can you fix that?

**Comment by lamblin** [comment]
Thanks!

**Pull request merged**